<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.38">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Sergio Lois">

<title>Publications</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<style>
    .quarto-title-block .quarto-title-banner {
      color: white;
background-image: url(../../title_banner.jpg);
background-size: cover;
    }
    </style>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../index.html">Portfolio</a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-about-me" role="button" data-bs-toggle="dropdown" aria-expanded="false">About me</a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-about-me">    
        <li>
    <a class="dropdown-item" href="../../about.html">
 <span class="dropdown-text">Who am I?</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../posts/thesis/mythesis.html">
 <span class="dropdown-text">My Thesis</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../../posts/publications/publications.html">
 <span class="dropdown-text">Publications</span></a>
  </li>  
    </ul>
  </li>
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="https://twitter.com/sergiolois">Twitter</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://www.linkedin.com/in/slois/">LinkedIn</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://github.com/slois">Github</a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-page-left">
      <h1 class="title">Publications</h1>
            <p class="subtitle lead">Genomics &amp; Bioinformatics</p>
                                <div class="quarto-categories">
                <div class="quarto-category">epigenetics</div>
                <div class="quarto-category">bioinformatics</div>
                <div class="quarto-category">genomics</div>
                <div class="quarto-category">methylome</div>
                <div class="quarto-category">transcriptome</div>
              </div>
                  </div>
  </div>
    
  
  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Sergio Lois <a href="https://orcid.org/0000-0002-6397-8693" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
            </div>
    </div>
      
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        
    </div>
<!-- main -->
<main class="content quarto-banner-title-block page-columns page-full column-page-left" id="quarto-document-content">




<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Prefoldin-like Bud27 influences the transcription of ribosomal components and ribosome biogenesis in Saccharomyces cerevisiae<span class="citation" data-cites="Martínez-Fernández2020">(<a href="#ref-Martínez-Fernández2020" role="doc-biblioref">Martínez-Fernández et al. 2020</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Understanding the functional connection that occurs for the three nuclear RNA polymerases to synthesize ribosome components during the ribosome biogenesis process has been the focal point of extensive research. To preserve correct homeostasis on the production of ribosomal components, cells might require the existence of proteins that target a common subunit of these RNA polymerases to impact their respective activities. This work describes how the yeast prefoldin-like Bud27 protein, which physically interacts with the Rpb5 common subunit of the three RNA polymerases, is able to modulate the transcription mediated by the RNA polymerase I, likely by influencing transcription elongation, the transcription of the RNA polymerase III, and the processing of ribosomal RNA. Bud27 also regulates both RNA polymerase II-dependent transcription of ribosomal proteins and ribosome biogenesis regulon genes, likely by occupying their DNA ORFs, and the processing of the corresponding mRNAs. With RNA polymerase II, this association occurs in a transcription rate-dependent manner. Our data also indicate that Bud27 inactivation alters the phosphorylation kinetics of ribosomal protein S6, a readout of TORC1 activity. We conclude that Bud27 impacts the homeostasis of the ribosome biogenesis process by regulating the activity of the three RNA polymerases and, in this way, the synthesis of ribosomal components. This quite likely occurs through a functional connection of Bud27 with the TOR signaling pathway.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
RNA-Seq Perspectives to Improve Clinical Diagnosis<span class="citation" data-cites="marco-puche2019">(<a href="#ref-marco-puche2019" role="doc-biblioref">Marco-Puche et al. 2019</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>In recent years, high-throughput next-generation sequencing technology has allowed a rapid increase in diagnostic capacity and precision through different bioinformatics processing algorithms, tools, and pipelines. The identification, annotation, and classification of sequence variants within different target regions are now considered a gold standard in clinical genetic diagnosis. However, this procedure lacks the ability to link regulatory events such as differential splicing to diseases. RNA-seq is necessary in clinical routine in order to interpret and detect among others splicing events and splicing variants, as it would increase the diagnostic rate by up to 10-35%. The transcriptome has a very dynamic nature, varying according to tissue type, cellular conditions, and environmental factors that may affect regulatory events such as splicing and the expression of genes or their isoforms. RNA-seq offers a robust technical analysis of this complexity, but it requires a profound knowledge of computational/statistical tools that may need to be adjusted depending on the disease under study. In this article we will cover RNA-seq analyses best practices applied to clinical routine, bioinformatics procedures, and present challenges of this approach.</p>

</div>
</div>
</div>
<div class="no-row-height column-margin column-container"><div class="callout-2-contents callout-collapse collapse callout-margin-content">
<p><img src="image5.png" class="img-fluid"> <img src="fgene-10-01152-g001.jpg" class="img-fluid"></p>
</div></div><div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Lineage specific transcription factors and epigenetic regulators mediate TGF β-dependent enhancer activation<span class="citation" data-cites="fueyo2018">(<a href="#ref-fueyo2018" role="doc-biblioref">Fueyo et al. 2018</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>During neurogenesis, dynamic developmental cues, transcription factors and histone modifying enzymes regulate the gene expression programs by modulating the activity of neural-specific enhancers. How transient developmental signals coordinate transcription factor recruitment to enhancers and to which extent chromatin modifiers contribute to enhancer activity is starting to be uncovered. Here, we take advantage of neural stem cells as a model to unravel the mechanisms underlying neural enhancer activation in response to the TGFβ signaling. Genome-wide experiments demonstrate that the proneural factor ASCL1 assists SMAD3 in the binding to a subset of enhancers. Once located at the enhancers, SMAD3 recruits the histone demethylase JMJD3 and the remodeling factor CHD8, creating the appropriate chromatin landscape to allow enhancer transcription and posterior gene activation. Finally, to analyze the phenotypical traits owed to cis-regulatory regions, we use CRISPR-Cas9 technology to demonstrate that the TGFβ-responsive Neurog2 enhancer is essential for proper neuronal polarization.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
The histone demethylase PHF8 is a molecular safeguard of the IFN γ response<span class="citation" data-cites="asensio-juan2017">(<a href="#ref-asensio-juan2017" role="doc-biblioref">Asensio-Juan et al. 2017</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>A precise immune response is essential for cellular homeostasis and animal survival. The paramount importance of its control is reflected by the fact that its non-specific activation leads to inflammatory events that ultimately contribute to the appearance of many chronic diseases. However, the molecular mechanisms preventing non-specific activation and allowing a quick response upon signal activation are not yet fully understood. In this paper we uncover a new function of PHF8 blocking signal independent activation of immune gene promoters. Affinity purifications coupled with mass spectrometry analysis identified SIN3A and HDAC1 corepressors as new PHF8 interacting partners. Further molecular analysis demonstrated that prior to interferon gamma (IFNγ) stimulation, PHF8 is bound to a subset of IFNγ-responsive promoters. Through the association with HDAC1 and SIN3A, PHF8 keeps the promoters in a silent state, maintaining low levels of H4K20me1. Upon IFNγ treatment, PHF8 is phosphorylated by ERK2 and evicted from the promoters, correlating with an increase in H4K20me1 and transcriptional activation. Our data strongly indicate that in addition to its well-characterized function as a coactivator, PHF8 safeguards transcription to allow an accurate immune response.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-5-contents" aria-controls="callout-5" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Cerebrospinal fluid-derived circulating tumour DNA better represents the genomic alterations of brain tumours than plasma<span class="citation" data-cites="demattos-arruda2015">(<a href="#ref-demattos-arruda2015" role="doc-biblioref">De Mattos-Arruda et al. 2015</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-5" class="callout-5-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Cell-free circulating tumour DNA (ctDNA) in plasma has been shown to be informative of the genomic alterations present in tumours and has been used to monitor tumour progression and response to treatments. However, patients with brain tumours do not present with or present with low amounts of ctDNA in plasma precluding the genomic characterization of brain cancer through plasma ctDNA. Here we show that ctDNA derived from central nervous system tumours is more abundantly present in the cerebrospinal fluid (CSF) than in plasma. Massively parallel sequencing of CSF ctDNA more comprehensively characterizes the genomic alterations of brain tumours than plasma, allowing the identification of actionable brain tumour somatic mutations. We show that CSF ctDNA levels longitudinally fluctuate in time and follow the changes in brain tumour burden providing biomarkers to monitor brain malignancies. Moreover, CSF ctDNA is shown to facilitate and complement the diagnosis of leptomeningeal carcinomatosis.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-6-contents" aria-controls="callout-6" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
A novel splice site mutation in the SERPING1 gene leads to haploinsufficiency by complete degradation of the mutant allele mRNA in a case of familial hereditary angioedema<span class="citation" data-cites="colobran2014">(<a href="#ref-colobran2014" role="doc-biblioref">Colobran, Pujol-Borrell, et al. 2014</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-6" class="callout-6-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Hereditary angioedema due to C1-inhibitor deficiency (HAE-C1INH) is a rare autosomal-dominant and life-threatening disorder caused by mutations in SERPING1 gene. It is characterized by attacks of angioedema involving the skin and/or the mucosa of the upper airways, as well as the intestinal mucosa. Here we report the case of a patient with HAE-C1INH without family history of angioedema. By sequencing the SERPING1 gene we detected a novel mutation (c.1249 + 5G &gt; A) affecting the 5’ donor splice site in intron 7. We analyzed the SERPING1 cDNA expecting a defect in splicing process but only the wild type allele was detected. SNP analysis of the cDNA sequence demonstrated that only one of the two alleles was present, indicating that the mRNA from the mutated allele was completely degraded. This study reinforces the concept of incomplete penetrance of this disorder since the patients’ mother never presented any sign of angioedema despite carrying the same mutation.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-7-contents" aria-controls="callout-7" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Novel and atypical splicing mutation in a compound heterozygous UNC13D defect presenting in Familial Hemophagocytic Lymphohistiocytosis triggered by EBV infection<span class="citation" data-cites="alsina2014">(<a href="#ref-alsina2014" role="doc-biblioref">Alsina et al. 2014</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-7" class="callout-7-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Familial Hemophagocytic Lymphohistiocytosis type 3 (FHL3) is a genetic disorder caused by mutations in UNC13D gene, coding the granule priming factor Munc13-4 that intervenes in NK and T cell cytotoxic function. Here we report the case of a 17-month-old girl with prolonged symptomatic EBV infectious mononucleosis and clinical symptoms of hemophagocytic syndrome. In vitro functional analysis pointed to a degranulation defect. The genetic analysis of UNC13D gene identified initially a heterozygous mutation (c.753+1G&gt;T) in the donor splice-site that resulted in exon 9 skipping (maternal allele). Mutations in other genes were considered, but additional analysis of UNC13D cDNA revealed in the paternal allele a heterozygous transition from G to A (c.2448-13G&gt;A) at the 3’ acceptor splice-site in intron 25, generating a new acceptor splice-site that leads to a frameshift and a premature STOP codon. Allele specific amplification of the cDNA confirmed the absence of a functional mRNA from the paternal allele. This case illustrates an atypical compound heterozygous UNC13D mutation affecting the RNA splicing that generates a typical FHL3 phenotype.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-8-contents" aria-controls="callout-8" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Identification and characterization of a novel splice site mutation in the SERPING1 gene in a family with hereditary angioedema<span class="citation" data-cites="colobran2014a">(<a href="#ref-colobran2014a" role="doc-biblioref">Colobran, Lois, et al. 2014</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-8" class="callout-8-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Hereditary angioedema due to C1-inhibitor deficiency (HAE-C1INH) is a rare autosomal-dominant disease caused by mutations in SERPING1 gene. The main clinical feature of C1INH deficiency is the spontaneous edema of the subcutaneous and submucosal layers. More than 280 different mutations scattering the entire SERPING1 gene have been reported. We identified and characterized a new mutation in SERPING1 gene in a Spanish family with hereditary angioedema. The mutation (c.685 + 2 T &gt; A) disrupts the donor splice site of intron 4 leading to the loss of exon 4 in mutant mRNA. We demonstrated that mutant mRNA is mostly degraded, probably by the surveillance pathway no-go mRNA decay. Bioinformatic analysis showed that the mutant protein, if produced, would be non-functional since the protein lacks a stretch of 45 amino acids affecting the functional RCL loop. Finally, we found a reduction of the wild-type mRNA expression in c.685 + 2 T &gt; A carriers.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-9-contents" aria-controls="callout-9" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Functional consequences of transferrin receptor-2 mutations causing hereditary hemochromatosis type 3<span class="citation" data-cites="joshi2015">(<a href="#ref-joshi2015" role="doc-biblioref">Joshi et al. 2015</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-9" class="callout-9-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Hereditary hemochromatosis (HH) type 3 is an autosomal recessive disorder of iron metabolism characterized by excessive iron deposition in the liver and caused by mutations in the transferrin receptor 2 (TFR2) gene. Here, we describe three new HH type 3 Spanish families with four TFR2 mutations (p.Gly792Arg, c.1606-8A&gt;G, Gln306*, and Gln672*). The missense variation p.Gly792Arg was found in homozygosity in two adult patients of the same family, and in compound heterozygosity in an adult proband that also carries a novel intronic change (c.1606-8A&gt;G). Two new nonsense TFR2 mutations (Gln306* and Gln672*) were detected in a pediatric case. We examine the functional consequences of two TFR2 variants (p.Gly792Arg and c.1606-8A&gt;G) using molecular and computational methods. Cellular protein localization studies using immunofluorescence demonstrated that the plasma membrane localization of p.Gly792Arg TFR2 is impaired. Splicing studies in vitro and in vivo reveal that the c.1606-8A&gt;G mutation leads to the creation of a new acceptor splice site and an aberrant TFR2 mRNA. The reported mutations caused HH type 3 by protein truncation, altering TFR2 membrane localization or by mRNA splicing defect, producing a nonfunctional TFR2 protein and a defective signaling transduction for hepcidin regulation. TFR2 genotyping should be considered in adult but also in pediatric cases with early-onset of iron overload.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-10-contents" aria-controls="callout-10" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
The Relationship between Gene Isoform Multiplicity, Number of Exons and Protein Divergence<span class="citation" data-cites="morata2013">(<a href="#ref-morata2013" role="doc-biblioref">Morata et al. 2013</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-10" class="callout-10-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>At present we know that phenotypic differences between organisms arise from a variety of sources, like protein sequence divergence, regulatory sequence divergence, alternative splicing, etc. However, we do not have yet a complete view of how these sources are related. Here we address this problem, studying the relationship between protein divergence and the ability of genes to express multiple isoforms. We used three genome-wide datasets of human-mouse orthologs to study the relationship between isoform multiplicity co-occurrence between orthologs (the fact that two orthologs have more than one isoform) and protein divergence. In all cases our results showed that there was a monotonic dependence between these two properties. We could explain this relationship in terms of a more fundamental one, between exon number of the largest isoform and protein divergence. We found that this last relationship was present, although with variations, in other species (chimpanzee, cow, rat, chicken, zebrafish and fruit fly). In summary, we have identified a relationship between protein divergence and isoform multiplicity co-occurrence and explained its origin in terms of a simple gene-level property. Finally, we discuss the biological implications of these findings for our understanding of inter-species phenotypic differences.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-11-contents" aria-controls="callout-11" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Prediction of pathological mutations in proteins: the challenge of integrating sequence conservation and structure stability principles<span class="citation" data-cites="riera2013">(<a href="#ref-riera2013" role="doc-biblioref">Riera, Lois, and Cruz 2013</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-11" class="callout-11-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The recent drop in genome sequencing costs has created a promising horizon for the development of genomic medicine. Within the biomedical environment, sequencing data are increasingly used for disease diagnosis and prognosis, treatment development, counseling, and so on. Many of these applications rely on the identification of disease causing variants. This is a particularly challenging problem because of the large number and wide variety of sequence variants identified in sequencing projects, and also because we only have a limited understanding of the physicochemical-biochemical properties that differentiate neutral from pathologic variants. Nonetheless, these last years have witnessed important methodological advances for one class of variants, those corresponding to changes in the amino-acid sequence of proteins.</p>
<p>Proteins are a main constituent of living systems. We know that although their biological properties are essentially determined by the amino-acid sequence, not all the changes in this sequence have the same impact. Some are neutral, but others affect protein function and lead to disease. A large body of evidence shows that whether one or the other is the case that depends on properties such as mutation location in the protein structure, interspecies conservation, and so on. Mutation prediction methods based on these features have good success rates, in the 70–90% range, although representation over time suggests there is a performance plateau that would limit their applicability. In light of the most recent advances in the field, and after reviewing the foundations of prediction methods, we discuss the existence of this performance threshold and how it can be overcomed.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-12-contents" aria-controls="callout-12" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Molecular damage in Fabry disease: characterization and prediction of alpha-galactosidase A pathological mutations<span class="citation" data-cites="riera2014">(<a href="#ref-riera2014" role="doc-biblioref">Riera et al. 2014</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-12" class="callout-12-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Loss-of-function mutations of the enzyme alpha-galactosidase A (GLA) causes Fabry disease (FD), that is a rare and potentially fatal disease. Identification of these pathological mutations by sequencing is important because it allows an early treatment of the disease. However, before taking any treatment decision, if the mutation identified is unknown, we first need to establish if it is pathological or not. General bioinformatic tools (PolyPhen-2, SIFT, Condel, etc.) can be used for this purpose, but their performance is still limited. Here we present a new tool, specifically derived for the assessment of GLA mutations. We first compared mutations of this enzyme known to cause FD with neutral sequence variants, using several structure and sequence properties. Then, we used these properties to develop a family of prediction methods adapted to different quality requirements. Trained and tested on a set of known Fabry mutations, our methods have a performance (Matthews correlation: 0.56-0.72) comparable or better than that of the more complex method, Polyphen-2 (Matthews correlation: 0.61), and better than those of SIFT (Matthews correl.: 0.54) and Condel (Matthews correl.: 0.51). This result is validated in an independent set of 65 pathological mutations, for which our method displayed the best success rate (91.0%, 87.7%, and 73.8%, for our method, PolyPhen-2 and SIFT, respectively). These data confirmed that our specific approach can effectively contribute to the identification of pathological mutations in GLA, and therefore enhance the use of sequence information in the identification of undiagnosed Fabry patients.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-13-contents" aria-controls="callout-13" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Overlapping DNA methylation dynamics in mouse intestinal cell differentiation and early stages of malignant progression<span class="citation" data-cites="forn2015">(<a href="#ref-forn2015" role="doc-biblioref">Forn et al. 2015</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-13" class="callout-13-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Mouse models of intestinal crypt cell differentiation and tumorigenesis have been used to characterize the molecular mechanisms underlying both processes. DNA methylation is a key epigenetic mark and plays an important role in cell identity and differentiation programs and cancer. To get insights into the dynamics of cell differentiation and malignant transformation we have compared the DNA methylation profiles along the mouse small intestine crypt and early stages of tumorigenesis. Genome-scale analysis of DNA methylation together with microarray gene expression have been applied to compare intestinal crypt stem cells (EphB2high), differentiated cells (EphB2negative), ApcMin/+ adenomas and the corresponding non-tumor adjacent tissue, together with small and large intestine samples and the colon cancer cell line CT26. Compared with late stages, small intestine crypt differentiation and early stages of tumorigenesis display few and relatively small changes in DNA methylation. Hypermethylated loci are largely shared by the two processes and affect the proximities of promoter and enhancer regions, with enrichment in genes associated with the intestinal stem cell signature and the PRC2 complex. The hypermethylation is progressive, with minute levels in differentiated cells, as compared with intestinal stem cells, and reaching full methylation in advanced stages. Hypomethylation shows different signatures in differentiation and cancer and is already present in the non-tumor tissue adjacent to the adenomas in ApcMin/+ mice, but at lower levels than advanced cancers. This study provides a reference framework to decipher the mechanisms driving mouse intestinal tumorigenesis and also the human counterpart.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-14-contents" aria-controls="callout-14" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells<span class="citation" data-cites="Jordà2017">(<a href="#ref-Jordà2017" role="doc-biblioref">Jordà et al. 2016</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-14" class="callout-14-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Cancer cells exhibit multiple epigenetic changes with prominent local DNA hypermethylation and widespread hypomethylation affecting large chromosomal domains. Epigenome studies often disregard the study of repeat elements owing to technical complexity and their undefined role in genome regulation. We have developed NSUMA (Next-generation Sequencing of UnMethylated Alu), a cost-effective approach allowing the unambiguous interrogation of DNA methylation in more than 130,000 individual Alu elements, the most abundant retrotransposon in the human genome. DNA methylation profiles of Alu repeats have been analyzed in colon cancers and normal tissues using NSUMA and whole-genome bisulfite sequencing. Normal cells show a low proportion of unmethylated Alu (1%-4%) that may increase up to 10-fold in cancer cells. In normal cells, unmethylated Alu elements tend to locate in the vicinity of functionally rich regions and display epigenetic features consistent with a direct impact on genome regulation. In cancer cells, Alu repeats are more resistant to hypomethylation than other retroelements. Genome segmentation based on high/low rates of Alu hypomethylation allows the identification of genomic compartments with differential genetic, epigenetic, and transcriptomic features. Alu hypomethylated regions show low transcriptional activity, late DNA replication, and its extent is associated with higher chromosomal instability. Our analysis demonstrates that Alu retroelements contribute to define the epigenetic landscape of normal and cancer cells and provides a unique resource on the epigenetic dynamics of a principal, but largely unexplored, component of the primate genome.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-15-contents" aria-controls="callout-15" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Deconstruction of DNA methylation patterns during myogenesis reveals specific epigenetic events in the establishment of the skeletal muscle lineage<span class="citation" data-cites="Carrió2015">(<a href="#ref-Carrió2015" role="doc-biblioref">carrió et al. 2015</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-15" class="callout-15-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The progressive restriction of differentiation potential from pluripotent embryonic stem cells (ESCs) to tissue-specific stem cells involves widespread epigenetic reprogramming, including modulation of DNA methylation patterns. Skeletal muscle stem cells are required for the growth, maintenance, and regeneration of skeletal muscle. To investigate the contribution of DNA methylation to the establishment of the myogenic program, we analyzed ESCs, skeletal muscle stem cells in proliferating (myoblasts) and differentiating conditions (myotubes), and mature myofibers. About 1.000 differentially methylated regions were identified during muscle-lineage determination and terminal differentiation, mainly located in gene bodies and intergenic regions. As a whole, myogenic stem cells showed a gain of DNA methylation, while muscle differentiation was accompanied by loss of DNA methylation in CpG-poor regions. Notably, the hypomethylated regions in myogenic stem cells were neighbored by enhancer-type chromatin, suggesting the involvement of DNA methylation in the regulation of cell-type specific enhancers. Interestingly, we demonstrated the hypomethylation of the muscle cell-identity Myf5 super-enhancer only in muscle cells. Furthermore, we observed that upstream stimulatory factor 1 binding to Myf5 super-enhancer occurs upon DNA demethylation in myogenic stem cells. Taken altogether, we characterized the unique DNA methylation signature of skeletal muscle stem cells and highlighted the importance of DNA methylation-mediated regulation of cell identity Myf5 super-enhancer during cellular differentiation.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-16-contents" aria-controls="callout-16" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Do protein motifs read the histone code?<span class="citation" data-cites="de2005">(<a href="#ref-de2005" role="doc-biblioref">Cruz et al. 2005</a>)</span>
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-16" class="callout-16-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The existence of different patterns of chemical modifications (acetylation, methylation, phosphorylation, ubiquitination and ADP-ribosylation) of the histone tails led, some years ago, to the histone code hypothesis. According to this hypothesis, these modifications would provide binding sites for proteins that can change the chromatin state to either active or repressed. Interestingly, some protein domains present in histone-modifying enzymes are known to interact with these covalent marks in the histone tails. This was first shown for the bromodomain, which was found to interact selectively with acetylated lysines at the histone tails. More recently, it has been described that the chromodomain can be targeted to methylation marks in histone N-terminal domains. Finally, the interaction between the SANT domain and histones is also well documented.</p>
<p>Overall, experimental evidence suggests that these domains could be involved in the recruitment of histone-modifying enzymes to discrete chromosomal locations, and/or in the regulation their enzymatic activity. Within this context, we review the distribution of bromodomains, chromodomains and SANT domains among chromatin-modifying enzymes and discuss how they can contribute to the translation of the histone code.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-17-contents" aria-controls="callout-17" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
The functional modulation of epigenetic regulators by alternative splicing<span class="citation" data-cites="Lois2007">(<a href="#ref-Lois2007" role="doc-biblioref">Sergio Lois et al. 2007</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-17" class="callout-17-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><strong>Background</strong>: Epigenetic regulators (histone acetyltransferases, methyltransferases, chromatin-remodelling enzymes, etc) play a fundamental role in the control of gene expression by modifying the local state of chromatin. However, due to their recent discovery, little is yet known about their own regulation. This paper addresses this point, focusing on alternative splicing regulation, a mechanism already known to play an important role in other protein families, e.g.&nbsp;transcription factors, membrane receptors, etc.</p>
<p><strong>Results</strong>: To this end, we compiled the data available on the presence/absence of alternative splicing for a set of 160 different epigenetic regulators, taking advantage of the relatively large amount of unexplored data on alternative splicing available in public databases. We found that 49 % (70 % in human) of these genes express more than one transcript. We then studied their alternative splicing patterns, focusing on those changes affecting the enzyme’s domain composition. In general, we found that these sequence changes correspond to different mechanisms, either repressing the enzyme’s function (e.g.&nbsp;by creating dominant-negative inhibitors of the functional isoform) or creating isoforms with new functions.</p>
<p><strong>Conclusion</strong>: We conclude that alternative splicing of epigenetic regulators can be an important tool for the function modulation of these enzymes. Considering that the latter control the transcriptional state of large sets of genes, we propose that epigenetic regulation of gene expression is itself strongly regulated by alternative splicing.</p>

</div>
</div>
</div>
<div class="no-row-height column-margin column-container"><div class="callout-17-contents callout-collapse collapse callout-margin-content">
<p><img src="image4.png" class="img-fluid"></p>
</div></div><div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-18-contents" aria-controls="callout-18" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Characterization of structural variability sheds light on the specificity determinants of the interaction between effector domains and histone tails<span class="citation" data-cites="Lois2010">(<a href="#ref-Lois2010" role="doc-biblioref">Sergi Lois et al. 2010</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-18" class="callout-18-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Structural characterization of the interaction between histone tails and effector modules (bromodomains, chromodomains, PHD fingers, etc.) is fundamental to understand the mechanistic aspects of epigenetic regulation of gene expression. In recent years many researchers have applied this approach to specific systems, thus providing a valuable but fragmentary view of the histone-effector interaction. In our work we use this information to characterize the structural features of the two main components of this interaction, histone peptides and the binding site of effector domains (focusing on those which target modified lysines), and increase our knowledge on its specificity determinants. Our results show that the binding sites of effectors are structurally variable, but some clear trends allow their classification in three main groups: flat-groove, narrow-groove and cavity-insertion. In addition, we found that even within these classes binding site variability is substantial. These results in context with the work from other researchers indicate that the there are at least two determinants of binding specificity in the binding site of effector modules. Finally, our analysis of the histone peptides sheds light on the structural transition experienced by histone tails upon effector binding, showing that it may vary depending on the local properties of the sequence stretch considered, thus allowing us to identify an additional specificity determinant for this interaction. Overall, the results of our analysis contribute to clarify the origins of specificity: different regions of the binding site and, in particular, differences in the disorder-order transitions experienced by different histone sequence stretches upon binding.</p>

</div>
</div>
</div>
<div class="no-row-height column-margin column-container"><div class="callout-18-contents callout-collapse collapse callout-margin-content">
<p><img src="image1.png" class="img-fluid"> <img src="image2.png" class="img-fluid"> <img src="image3.png" class="img-fluid"></p>
</div></div><div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-19-contents" aria-controls="callout-19" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Preservation of protein clefts in comparative models<span class="citation" data-cites="Piedra2008">(<a href="#ref-Piedra2008" role="doc-biblioref">Piedra, Lois, and Cruz 2008</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-19" class="callout-19-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><strong>Background</strong>: Comparative, or homology, modelling of protein structures is the most widely used prediction method when the target protein has homologues of known structure. Given that the quality of a model may vary greatly, several studies have been devoted to identifying the factors that influence modelling results. These studies usually consider the protein as a whole, and only a few provide a separate discussion of the behaviour of biologically relevant features of the protein. Given the value of the latter for many applications, here we extended previous work by analysing the preservation of native protein clefts in homology models. We chose to examine clefts because of their role in protein function/structure, as they are usually the locus of protein-protein interactions, host the enzymes’ active site, or, in the case of protein domains, can also be the locus of domain-domain interactions that lead to the structure of the whole protein.</p>
<p><strong>Results</strong>: We studied how the largest cleft of a protein varies in comparative models. To this end, we analysed a set of 53507 homology models that cover the whole sequence identity range, with a special emphasis on medium and low similarities. More precisely we examined how cleft quality - measured using six complementary parameters related to both global shape and local atomic environment, depends on the sequence identity between target and template proteins. In addition to this general analysis, we also explored the impact of a number of factors on cleft quality, and found that the relationship between quality and sequence identity varies depending on cleft rank amongst the set of protein clefts (when ordered according to size), and number of aligned residues.</p>
<p><strong>Conclusion</strong>: We have examined cleft quality in homology models at a range of seq.id. levels. Our results provide a detailed view of how quality is affected by distinct parameters and thus may help the user of comparative modelling to determine the final quality and applicability of his/her cleft models. In addition, the large variability in model quality that we observed within each sequence bin, with good models present even at low sequence identities (between 20% and 30%), indicates that properly developed identification methods could be used to recover good cleft models in this sequence range.</p>
</div>
</div>
</div>
<div class="callout-note callout callout-style-default callout-captioned">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-20-contents" aria-controls="callout-20" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-caption-container flex-fill">
Characterization of the impact of alternative splicing on protein dynamics: The cases of glutathione S- transferase and ectodysplasin-A isoforms<span class="citation" data-cites="Barbany2012">(<a href="#ref-Barbany2012" role="doc-biblioref">Barbany et al. 2012</a>)</span>.
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-20" class="callout-20-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Recent studies have shown how alternative splicing (AS), the process by which eukaryotic genes express more than one product, affects protein sequence and structure. However, little information is available on the impact of AS on protein dynamics, a property fundamental for protein function. In this work, we have addressed this issue using molecular dynamics simulations of the isoforms of two model proteins: glutathione S-transferase and ectodysplasin-A. We have found that AS does not have a noticeable impact on global or local structure fluctuations. We have also found that, quite interestingly, AS has a significant effect on the coupling between key structural elements such as surface cavities. Our results provide the first atom-level view of the impact of AS on protein dynamics, as far as we know. They can contribute to refine our present view of the relationship between AS and protein disorder and, more importantly, they reveal how AS may modify structural dynamic couplings in proteins.</p>
</div>
</div>
</div>




<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-alsina2014" class="csl-entry" role="doc-biblioentry">
Alsina, L., R. Colobran, M. F. de Sevilla, A. Català, L. Viñas, S. Ricart, A. M. Plaza, et al. 2014. <span>“Novel and Atypical Splicing Mutation in a Compound Heterozygous Unc13d Defect Presenting in Familial Hemophagocytic Lymphohistiocytosis Triggered by EBV Infection.”</span> <em>Clinical Immunology</em> 153 (2): 292–97. <a href="https://doi.org/10.1016/j.clim.2014.04.019">https://doi.org/10.1016/j.clim.2014.04.019</a>.
</div>
<div id="ref-asensio-juan2017" class="csl-entry" role="doc-biblioentry">
Asensio-Juan, Elena, Raquel Fueyo, Stella Pappa, Simona Iacobucci, Carmen Badosa, Sergi Lois, Miriam Balada, et al. 2017. <span>“The Histone Demethylase Phf8 Is a Molecular Safeguard of the IFNγ Response.”</span> <em>Nucleic Acids Research</em>, January, gkw1346. <a href="https://doi.org/10.1093/nar/gkw1346">https://doi.org/10.1093/nar/gkw1346</a>.
</div>
<div id="ref-Barbany2012" class="csl-entry" role="doc-biblioentry">
Barbany, Montserrat, Jordi Morata, Tim Meyer, Sergi Lois, Modesto Orozco, and Xavier de la Cruz. 2012. <span>“Characterization of the Impact of Alternative Splicing on Protein Dynamics: The Cases of Glutathione S-Transferase and Ectodysplasin-A Isoforms.”</span> <em>Proteins: Structure, Function, and Bioinformatics</em> 80 (9): 2235–49. <a href="https://doi.org/10.1002/prot.24112">https://doi.org/10.1002/prot.24112</a>.
</div>
<div id="ref-Carrió2015" class="csl-entry" role="doc-biblioentry">
carrió, Elvira, Anna Díez-Villanueva, Sergi Lois, Izaskun Mallona, Ildefonso Cases, Marta Forn, Miguel A. Peinado, and Mònica Suelves. 2015. <span>“Deconstruction of DNA Methylation Patterns During Myogenesis Reveals Specific Epigenetic Events in the Establishment of the Skeletal Muscle Lineage.”</span> <em>Stem Cells</em> 33 (6): 2025–36. <a href="https://doi.org/10.1002/stem.1998">https://doi.org/10.1002/stem.1998</a>.
</div>
<div id="ref-colobran2014a" class="csl-entry" role="doc-biblioentry">
Colobran, Roger, Sergio Lois, Xavier de la Cruz, Ricardo Pujol-Borrell, Manuel Hernández-González, and Mar Guilarte. 2014. <span>“Identification and Characterization of a Novel Splice Site Mutation in the Serping1 Gene in a Family with Hereditary Angioedema.”</span> <em>Clinical Immunology</em> 150 (2): 143–48. <a href="https://doi.org/10.1016/j.clim.2013.11.013">https://doi.org/10.1016/j.clim.2013.11.013</a>.
</div>
<div id="ref-colobran2014" class="csl-entry" role="doc-biblioentry">
Colobran, Roger, Ricardo Pujol-Borrell, Manuel Hernández-González, and Mar Guilarte. 2014. <span>“A Novel Splice Site Mutation in the Serping1 Gene Leads to Haploinsufficiency by Complete Degradation of the Mutant Allele mRNA in a Case of Familial Hereditary Angioedema.”</span> <em>Journal of Clinical Immunology</em> 34 (5): 521–23. <a href="https://doi.org/10.1007/s10875-014-0042-3">https://doi.org/10.1007/s10875-014-0042-3</a>.
</div>
<div id="ref-de2005" class="csl-entry" role="doc-biblioentry">
Cruz, Xavier de la, Sergio Lois, Sara Sánchez-Molina, and Marian A. Martínez-Balbás. 2005. <span>“Do Protein Motifs Read the Histone Code?”</span> <em>BioEssays</em> 27 (2): 164–75. <a href="https://doi.org/10.1002/bies.20176">https://doi.org/10.1002/bies.20176</a>.
</div>
<div id="ref-demattos-arruda2015" class="csl-entry" role="doc-biblioentry">
De Mattos-Arruda, Leticia, Regina Mayor, Charlotte K. Y. Ng, Britta Weigelt, Francisco Martínez-Ricarte, Davis Torrejon, Mafalda Oliveira, et al. 2015. <span>“Cerebrospinal Fluid-Derived Circulating Tumour DNA Better Represents the Genomic Alterations of Brain Tumours Than Plasma.”</span> <em>Nature Communications</em> 6 (1). <a href="https://doi.org/10.1038/ncomms9839">https://doi.org/10.1038/ncomms9839</a>.
</div>
<div id="ref-forn2015" class="csl-entry" role="doc-biblioentry">
Forn, Marta, Anna Díez-Villanueva, Anna Merlos-Suárez, Mar Muñoz, Sergi Lois, Elvira Carriò, Mireia Jordà, Anna Bigas, Eduard Batlle, and Miguel A. Peinado. 2015. <span>“Overlapping DNA Methylation Dynamics in Mouse Intestinal Cell Differentiation and Early Stages of Malignant Progression.”</span> Edited by Jorg Tost. <em>PLOS ONE</em> 10 (5): e0123263. <a href="https://doi.org/10.1371/journal.pone.0123263">https://doi.org/10.1371/journal.pone.0123263</a>.
</div>
<div id="ref-fueyo2018" class="csl-entry" role="doc-biblioentry">
Fueyo, Raquel, Simona Iacobucci, Stella Pappa, Conchi Estarás, Sergio Lois, Marta Vicioso-Mantis, Claudia Navarro, et al. 2018. <span>“Lineage Specific Transcription Factors and Epigenetic Regulators Mediate TGFβ-Dependent Enhancer Activation.”</span> <em>Nucleic Acids Research</em> 46 (7): 3351–65. <a href="https://doi.org/10.1093/nar/gky093">https://doi.org/10.1093/nar/gky093</a>.
</div>
<div id="ref-Jordà2017" class="csl-entry" role="doc-biblioentry">
Jordà, Mireia, Anna Díez-Villanueva, Izaskun Mallona, Berta Martín, Sergi Lois, Víctor Barrera, Manel Esteller, Tanya Vavouri, and Miguel A. Peinado. 2016. <span>“The Epigenetic Landscape of <span><em>Alu</em></span> Repeats Delineates the Structural and Functional Genomic Architecture of Colon Cancer Cells.”</span> <em>Genome Research</em> 27 (1): 118–32. <a href="https://doi.org/10.1101/gr.207522.116">https://doi.org/10.1101/gr.207522.116</a>.
</div>
<div id="ref-joshi2015" class="csl-entry" role="doc-biblioentry">
Joshi, Ricky, Maya Shvartsman, Erica Morán, Sergi Lois, Jessica Aranda, Anna Barqué, Xavier Cruz, et al. 2015. <span>“Functional Consequences of Transferrin Receptor<span>-</span>2 Mutations Causing Hereditary Hemochromatosis Type 3.”</span> <em>Molecular Genetics &amp; Genomic Medicine</em> 3 (3): 221–32. <a href="https://doi.org/10.1002/mgg3.136">https://doi.org/10.1002/mgg3.136</a>.
</div>
<div id="ref-Lois2010" class="csl-entry" role="doc-biblioentry">
Lois, Sergi, Naiara Akizu, Gemma Mas de Xaxars, Iago Vázquez, Marian Martínez-Balbás, and Xavier de la Cruz. 2010. <span>“Characterization of Structural Variability Sheds Light on the Specificity Determinants of the Interaction Between Effector Domains and Histone Tails.”</span> <em>Epigenetics</em> 5 (2): 137–48. <a href="https://doi.org/10.4161/epi.5.2.11079">https://doi.org/10.4161/epi.5.2.11079</a>.
</div>
<div id="ref-Lois2007" class="csl-entry" role="doc-biblioentry">
Lois, Sergio, Noemí Blanco, Marian Martínez-Balbás, and Xavier de la Cruz. 2007. <span>“The Functional Modulation of Epigenetic Regulators by Alternative Splicing.”</span> <em>BMC Genomics</em> 8 (1). <a href="https://doi.org/10.1186/1471-2164-8-252">https://doi.org/10.1186/1471-2164-8-252</a>.
</div>
<div id="ref-marco-puche2019" class="csl-entry" role="doc-biblioentry">
Marco-Puche, Guillermo, Sergio Lois, Javier Benítez, and Juan Carlos Trivino. 2019. <span>“RNA-Seq Perspectives to Improve Clinical Diagnosis.”</span> <em>Frontiers in Genetics</em> 10 (November). <a href="https://doi.org/10.3389/fgene.2019.01152">https://doi.org/10.3389/fgene.2019.01152</a>.
</div>
<div id="ref-Martínez-Fernández2020" class="csl-entry" role="doc-biblioentry">
Martínez-Fernández, Verónica, Abel Cuevas-Bermúdez, Francisco Gutiérrez-Santiago, Ana I. Garrido-Godino, Olga Rodríguez-Galán, Antonio Jordán-Pla, Sergio Lois, Juan C. Triviño, Jesús de la Cruz, and Francisco Navarro. 2020. <span>“Prefoldin-Like Bud27 Influences the Transcription of Ribosomal Components and Ribosome Biogenesis in <span><em>Saccharomyces Cerevisiae</em></span>.”</span> <em>RNA</em> 26 (10): 1360–79. <a href="https://doi.org/10.1261/rna.075507.120">https://doi.org/10.1261/rna.075507.120</a>.
</div>
<div id="ref-morata2013" class="csl-entry" role="doc-biblioentry">
Morata, Jordi, Santi Béjar, David Talavera, Casandra Riera, Sergio Lois, Gemma Mas de Xaxars, and Xavier de la Cruz. 2013. <span>“The Relationship Between Gene Isoform Multiplicity, Number of Exons and Protein Divergence.”</span> Edited by Gil Ast. <em>PLoS ONE</em> 8 (8): e72742. <a href="https://doi.org/10.1371/journal.pone.0072742">https://doi.org/10.1371/journal.pone.0072742</a>.
</div>
<div id="ref-Piedra2008" class="csl-entry" role="doc-biblioentry">
Piedra, David, Sergi Lois, and Xavier de la Cruz. 2008. <span>“Preservation of Protein Clefts in Comparative Models.”</span> <em>BMC Structural Biology</em> 8 (1). <a href="https://doi.org/10.1186/1472-6807-8-2">https://doi.org/10.1186/1472-6807-8-2</a>.
</div>
<div id="ref-riera2013" class="csl-entry" role="doc-biblioentry">
Riera, Casandra, Sergio Lois, and Xavier de la Cruz. 2013. <span>“Prediction of Pathological Mutations in Proteins: The Challenge of Integrating Sequence Conservation and Structure Stability Principles.”</span> <em>Wiley Interdisciplinary Reviews: Computational Molecular Science</em> 4 (3): 249–68. <a href="https://doi.org/10.1002/wcms.1170">https://doi.org/10.1002/wcms.1170</a>.
</div>
<div id="ref-riera2014" class="csl-entry" role="doc-biblioentry">
Riera, Casandra, Sergio Lois, Carmen Domínguez, Israel Fernandez-Cadenas, Joan Montaner, Victor Rodríguez-Sureda, and Xavier de la Cruz. 2014. <span>“Molecular Damage in Fabry Disease: Characterization and Prediction of Alpha-Galactosidase A Pathological Mutations.”</span> <em>Proteins: Structure, Function, and Bioinformatics</em> 83 (1): 91–104. <a href="https://doi.org/10.1002/prot.24708">https://doi.org/10.1002/prot.24708</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
      <div class="nav-footer-center">Copyright 2022, Sergio Lois</div>
  </div>
</footer>



</body></html>